pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[project]
```

Comment:
Should we put authors in here? And license?

Reply:
I don't want to maintain the author list

The license is fairly minor though